### PR TITLE
Emacs: Preserve behavior of M-<arrows> in org-mode

### PR DESCRIPTION
**Pull request by vmalloc**

### CODE REVIEW

File: emacs.d/init.el [hunk 1]
```diff
@@ -221,7 +221,12 @@
 (use-package org
   :bind (("C-M-<return>" . #'org-insert-heading-after-current)
          ("<f10>" . org-agenda)
-         ("C-<f10>" . org-capture))
+         ("C-<f10>" . org-capture)
+         :map org-mode-map
+         ("M-<right>" . nil)
```
Comment (drrlvn) [inline emacs.d/init.el]: Do these appear in `personal-keybindings`? Try `describe-personal-keybindings` and see.

If they do then I think that the preferred way is calling `unbind-key` in the `:config` section of the `use-package`.

Comment (vmalloc) [reply emacs.d/init.el]: @drrlvn nope, they appear in `init.el`, with the `use-package` of `windmove`...

Comment (vmalloc) [reply emacs.d/init.el]: Oh sorry, I misunderstood

**Comment by vmalloc** [comment]
Fixed and pushed again

**Pull request merged**